artemis-server/src/main/java/org/apache/activemq/artemis/core/security/impl/SecurityStoreImpl.java

```diff
@@ -424,26 +432,71 @@ private void handleNoCacheLoginException(NoCacheLoginException e) {
       logger.debug("Skipping authentication cache due to exception: {}", e.getMessage());
    }
 
+   private void putAuthenticationCacheEntry(String user,
+                                            String password,
+                                            RemotingConnection connection,
+                                            Subject subject) {
+      if (authenticationCache != null) {
+         authenticationCache.put(createAuthenticationCacheKey(user, password, connection), new Pair<>(subject != null, subject));
+      }
+   }
+
+   private Pair<Boolean, Subject> getAuthenticationCacheEntry(String user,
+                                                              String password,
+                                                              RemotingConnection connection) {
+      if (authenticationCache == null) {
+         return null;
+      } else {
+         return authenticationCache.getIfPresent(createAuthenticationCacheKey(user, password, connection));
+      }
+   }
+
+   private void getAuthorizationCacheEntry(ConcurrentHashSet<SimpleString> set, String key) {
```

Comment:
Think this should be putAuthorizationCacheEntry

Reply:
Indeed!